### DISCUSSION TITLE
Please add 'animation delay'

### DISCUSSION THREAD

**Discussion by IamLizu** [tailwindlabs/tailwindcss#6144] [Ideas]
In regular CSS, we have `animation-delay` which as it suggest can delay the animation to the applied element. In current docs of [animation](https://tailwindcss.com/docs/animation), we don't see any support for that.

**Comment by adamwathan** [comment]
Hey! Can you share an example of when you've needed it alongside our existing animation support or of exactly what you are trying to achieve just to make sure if we implement it we do it in the right way to solve your problem?

**Reply by IamLizu** [reply]
```
import React from "react";
import NonTargetedModal from "../components/modals/NonTargeted";

export default function Test() {
    return (
        <NonTargetedModal>
            <div className="flex justify-center space-x-1 font-semibold text-gray-500 text-4xl">
                <p className="animate-bounce">.</p>
                <p className="animate-bounce">.</p>
                <p className="animate-bounce">.</p>
            </div>
        </NonTargetedModal>
    );
}
```
Above is the code of loading component. The component renders a component where the screen has only three bouncing dots(.). They are bouncing at the same speed. 

What I want is each dot to bounce at different speed so what it would create more like a wave. If there was this feature, I would do it like this for example,
```
<p className="animate-bounce animation-delay-10">.</p>
<p className="animate-bounce animation-delay-20">.</p>
<p className="animate-bounce animation-delay-30">.</p>
```
I hope this clarifies what I am saying. I would be happy to create a sandbox to show what I am saying if not already clear.

**Comment by sarukomine** [comment]
I want it too, but i think Tailwind team could do more in animation part, not just `animation-delay`, so the team should split it to more classes as below.

for animation duration:
- animation-duration-75
- animation-duration-100
- animation-duration-150
- ...

for animation timing function:
- animation-linear
- animation-ease
- animation-ease-in
- ...

for animation delay:
- animation-delay-75
- animation-delay-100
- animation-delay-150
- ...

for animation iteration count:
- animation-infinite
- animation-once
- animation-twice
- animation-thrice

or more...

we also could use these:
- animation-[spin]
- animation-duration-[2s]
- animation-delay-[2s]
- animation-delay-[200ms]
